pathogen_token/rewards.sol

```diff
@@ -1,91 +1,77 @@
 pragma solidity ^0.5.0;
 
-    /**
-    * @notice The accumulated rewards for each Token.
-    */
+    /*The accumulated rewards for each Token */
     
+    /* A rewards mapping object is created */
     mapping(address => uint256) internal rewards;
     
-    /**
-    * @notice A method to allow  pathogen to check his rewards.
-    * @param _pathogen The pathogen to check rewards for.
-    */
-    
-    function rewardOf(address _pathogen)
+    /* Define a function that will take an address and
+    pass it to the rewards mapping object. Any stakeholder
+    can check their rewards in this function */
+
+    function check_rewards(address _pathogen)
         public
         view
-        returns(uint256)
-        
+        returns(uint256)      
     {
-        return rewards(_pathogen);
-        
-    }
-    
-    
-    /**
-    * @notice A method to the aggregated rewards from all Tokens.
-    * @return uint256 The aggregated rewards from all Tokens.
-    */
-    
-    function totalRewards()
+        return rewards(_pathogen);      
+    }  
+
+    /* Calcualte the aggregated rewards from all tokens
+    and return the aggregated rewards */
+
+    function total_rewards()
         public
         view
-        returns(uint256)
-        
+        returns(uint256)    
     {
-        uint256 _totalRewards = 0;
-        for (uint256 t = 0; s < pathogen.length t += 1){
-            _totalRewards = _totalRewards.add(rewards[pathogen[t]]);
+        /* the reward for the pathogen token is 0.2 percent
+        use path_reward as the variable to iterate over the pathogen array */
+
+        uint256 _aggregated_reward = 0;
+        for (uint256 path_reward = 0; path_reward < pathogen.length; path_reward += 0.2)
+        {
+            _aggregated_reward = _aggregated_reward.add(rewards[pathogen[path_reward]]);
             
-        }
-        
-        return _totalRewards;
+        }     
+        return _aggregated_reward;
     }
         
-    /**
-    * @notice A simple method that calculates the rewards for each Token.
-    * @param _pathogen The pathogen to calculate rewards for.
-    */
+    /* calculate the rewards for each Token */
     
-    function calculateReward(address _pathogen)
+    function calculate_reward(address _pathogen)
         public
         view
-        returns(uint256)
-        
+        returns(uint256)  
     {
-        returns Tokens(_pathogen) /100;
+        return stake_record(_pathogen) /100;
         
-    }
+    } 
     
-    /**
-    * @notice A method to distribute rewards to all stakeholders.
-    */
-    function distributeRewards()
-        public
-        onlyOwner
-        
+    /* Distribute rewards to all Tokens 
+    @steven: this is confusing. We iterate over the length of the pathogen array to add 
```

Comment:
Lets try to keep the comments more in github comments, and not within the code as this is public and will not understand your direct reference.